server/channels/app/platform/web_hub.go

```diff
@@ -556,7 +557,7 @@ func (h *Hub) Start() {
 				}
 
 				conns := connIndex.ForUser(webConn.UserId)
-				if len(conns) == 0 || areAllInactive(conns) {
```

Comment:
Perhaps we could add a comment here as well, or simply rename `areAllInactive` because the logic change is not obvious from here.